portfolio/src/main/webapp/index.html

```diff
@@ -21,7 +20,30 @@ <h2>About Me</h2>
             minoring in Philsophy and Consumer Psychology. This summer I am a Google Step intern
             looking to gain insight into a potential career path whilst developing my professional
             etiquette. </p>
-        <div id="center"> <img src="images/tyler.jpg" alt="Flowerboy Album Cover"> </div>
+        <p>I went to a choir school for three years where I was able to cultivate my creative 
+            interests. After that I finished my high school career at an all boys boarding school 
+            where sports were highly popular. These two experiences inspired the work I will be sharing
+            in this portfolio. Flowerboy is my account of what Tyler The Creator's album "Flowerboy" 
+            means to me and the impression it has had. Ballers is an image grid of my soccer interests
+            with the teams/players that have been most influential for me. </p>
+    </div>
+    <div>
+        <h2>Flowerboy</h2>
+        <p> Amongst Tyler's albums my easy going demeanour is best matched by Flowerboy. Before the first 
+            note is sound the album cover depicts a vivid scenery which is oddly soothing. The bright 
+            background contrasts the figure standing with folded arms whilst facing away from the viewer. 
+            The massive bees show that this is not realistic and verify the attractiveness of the Flowerboy. 
+            I imagine that Tyler gave us a glimpse into his extraordinary imagination without allowing us 
+            to loose sense of who his calm character. The music continues to build the narrative he portrayed
+            on the cover by by mixing a variety of songs which blend to share Tyler's current views. He is
+            somehow composed on this album whilst discussing his depression, lonliness and love stories. 
+            The legitimacy of his sentiment is conveyed in his unconventional chord progressions and ertaic
+            developments. By not allowing us to know what comes next Tyler makes us experience his unstelled 
+            feelings. The style is uniquely his own with inspiration drawn from the many artists he features. 
+            This album remains amongst my favourite listens and I could play See You Again on repeat for 
+            hours. I like to think that I could be described as a Flowerboy, both expressive and calm </p> 
+        <div id="center"> 
+            <img src="images/tyler.jpg" width="700" height="700" alt="Flowerboy Album Cover"> </div> 
```

Comment:
1. Try to use two spaces indentation for this file. 
2. Move "</div>" to a new line as style guide says: "Use a new line for every block, list, or table element, and indent every such child element."